lib/redirect_map.rb

```diff
@@ -0,0 +1,23 @@
+require 'yaml'
+require 'singleton'
+
+class RedirectMap
+  include Singleton
+
+  attr_reader :redirects
```

Comment:
Why? Worry that giving direct access to the backing hash, instead of going through the method, isn't want we want...